libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module ChefAnalyticsCookbook
+  module Helpers
+    def api_fqdn_available?
+      return false if node['chef-analytics'].nil?
+      return false if node['chef-analytics']['api_fqdn'].nil?
+      !node['chef-analytics']['api_fqdn'].empty?
```

Comment:
Right, but its more idiomatic to use the expression I posted there until 2.3 ships --  the `return false` takes me longer to read the code because i wind up reading that and making certain all the edge conditions match the `&&` chaining version or not.